File: app-pacientes-backend/src/main/java/pe/tintegro/apppacientesbackend/model/TipoDocumentoIdentidad.java
package pe.tintegro.apppacientesbackend.model;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class TipoDocumentoIdentidad {

    private Integer idTipoDocIdentidad;
    private String descripcion;
    private String codigoIeds;
    private Integer estado;

}

File: app-pacientes-frontend/src/app/data/types/TipoDocumentoIdentidad.ts
export class TipoDocumentoIdentidad {
  idTipoDocIdentidad: number;
  descripcion: string;
  codigoIeds: string;

  constructor(
    idTipoDocIdentidad: number = 0,
    descripcion: string = '',
    codigoIeds: string = ''
  ) {
    this.idTipoDocIdentidad = idTipoDocIdentidad;
    this.descripcion = descripcion;
    this.codigoIeds = codigoIeds;
  }
}